lib/chef-acceptance/suite_runner.rb

```diff
@@ -0,0 +1,94 @@
+require 'chef-acceptance/output_formatter'
+require 'chef-acceptance/chef_runner'
+
+module ChefAcceptance
+  # Responsible for running one or more suites and displaying statistics about
+  # the runs to the user
+  class SuiteRunner
+    attr_reader :force_destroy
+
+    def initialize(force_destroy = false)
+      @force_destroy = force_destroy
+    end
+
+    def run(suite_regex, command)
+      output_formatter = OutputFormatter.new
+      suite_stats = SuiteStatistics.new
+      suites = find_suites(suite_regex)
+      suites.each do |suite|
+        if command == "test"
+          commands = AcceptanceCookbook::CORE_ACCEPTANCE_RECIPES
+        else
+          commands = [command]
+        end
+
+        test_suite = TestSuite.new(suite)
+        runner = ChefRunner.new(test_suite)
+        commands.each do |command|
+          command_stats = runner.run(command)
```

Comment:
I think we have a bit too much passing around for the stats. 

Since we are keeping time for each suite and each phase, I would measure it here instead of inside the ShelloutBuilder. Instead of having a `SuiteStatistics` class we can have a singleton `Stats` class which we can call to add errors and timing information that we calculate here. I think that will be simpler because we will not be depending on the return values.